.storybook/main.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   stories: [
     '../app/component-library/components/**/*.stories.?(ts|tsx|js|jsx)',
+    '../app/component-library/components-temp/TagColored/**/*.stories.?(ts|tsx|js|jsx)',
```

Comment:
I won't block you here, but I probably would have made this more universal by not using `TagColored` and utilizing the globstar pathname 
```suggestion
    '../app/component-library/components-temp/**/*.stories.?(ts|tsx|js|jsx)',
```

Reply:
Don't worry about this as much. I know this is just a temp thing